USER_MANUAL.md

```diff
@@ -1549,58 +1729,912 @@ The following figure offers a visual guideline for the various parameters that c
+- [Source File](#duration-source-file)
+
+#### non_repairable_multiple_emissions_per_source (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [multiple_emissions_per_source](#multiple_emissions_per_source-propagating-parameter) for more details on what this parameter represents. In this file, this is a `TRUE`/`FALSE` value, representing if there can be multiple emissions existing at a given time for non-repairable emissions for the given site.
+
+This parameter can also be set in the following files:
+
+- [Virtual World Parameter file](#multiple_emissions_per_source-propagating-parameter)
+- [Site type File](#non_repairable_multiple_emissions_per_source-site-type-file)
+- [Equipment File](#non_repairable_multiple_emissions_per_source-equipment-file)
+- [Source File](#multiple_emissions_per_source-source-file)
+
+#### {method}_surveys_per_year (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See[surveys_per_year](#surveys_per_year-propagating-parameter) for more details on what this parameter represents. In this file, this is a single integer representing the number of surveys to be conducted in a given year for the specified site for the specified method.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#surveys_per_year-propagating-parameter)
+- [Site type File](#method_surveys_per_year-site-type-file)
+
+#### {method}_deploy_years (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See [deploy_year](#deployment_years--propagating-parameter) for more details on what this parameter represents. In this file, this is a list of integers representing which years the specified method can be deployed on.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#deployment_years--propagating-parameter)
+- [Site type File](#method_deploy_years-site-type-file)
+
+#### {method}_deploy_months (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See[deploy_month](#deployment_months--propagating-parameter) for more details on what this parameter represents. In this file, this is a list of integers representing which months the specified method can be deployed on.
+
+This parameter can also be set in the following files:
+
+- [Method Parameter file](#deployment_months--propagating-parameter)
+- [Site type File](#method_deploy_months-site-type-file)
+
+#### {method}_spatial (Sites file)
+
+**Description:** This is a propagating parameter that can be set at multiple levels of granularity. See[spatial](#spatial-propagating-parameter) for more details on what this parameter represents. In this file, this is a single value, representing the chance in which the given method can detect emissions at the given site.
```

Comment:
I don't feel great about this one but I don't have a suggestion on how to improve